Client/Program.cs

```diff
@@ -47,9 +52,36 @@
     {
         cp.SetIsOriginAllowed(origin => new Uri(origin).IsLoopback);
     });
-});
-
+});
+
 #region ConfigureServices
+
+builder.Services.AddLocalization(options =>
```

Comment:
To support webasm, this configuration should probably be set in ClientUI.Shared/Extensions/ServiceCollectionExtensions